README.md

```diff
@@ -6,9 +6,9 @@ This repository is used to assemble the portable (or all-in-one) packages of the
 
 If you'd like to contribute to the Open Hospital project, please read [CONTRIBUTING.md][contributing].
 
-## How to create Open Hospital packages
+## How to create Portable Open Hospital packages
 
-To create the Open Hospital packages,
+To create the Portable Open Hospital packages,
```

Comment:
actually, the script also builds what @mwithi called the "Full" distribution, which doesn't include MySQL and JRE.

Reply:
Corrected. To be consistent we should also change the name of the script though, it's called poh.sh....
I find it a little bit confusing....

Reply:
@mizzioisf you mean **build_poh.sh**, right? I think yes we could just rename it to `build_release.sh` since is what is actually doing.

Reply:
Absolutely agree on "build_release.sh".

Reply:
that script will be replaced by a more robust Makefile in #135

Reply:
> that script will be replaced by a more robust Makefile in #135

yeah true, I was going to mention it! thanks!